### ISSUE TITLE
How can i set width and height to 100%?

### ISSUE THREAD

**Issue by Konstantinov-Innokentii** [bokuweb/react-rnd#146]
Hi
How can i set width and height to 100% ? In docs said : "For example, you can set 300, '300px', 50%", but with percents it doesn't working. Instead i have 100px, but no 100%. U can see code below. 
I am using react-rnd: 5.0.8 and Chromium browser.
```
 default={{

                     x: this.props.window.x,
                     y: this.props.window.y,
                     width: '100%',
                     height: '100%',

                 }}
```
Lot of thanks.

**Comment by sajadghawami** [comment]
this is a workaround:

You create a div with `100%` for width and height, and create a `ref`:

```javascript
pageSize;

render() {
    return (
        <div style={{width: '100%', height:'100%'}} ref={(pageSize) => this.pageSize = pageSize}>
        </div>
    )
}
```

Now, when calling `this.pageSize.clientWidth` or `this.pageSize.clientHeight` you will get the sizes in pixel.

**Comment by bokuweb** [comment]
Fixed y v6.00.

Please try as follows.

```
<Rnd
  width={100%}
  height={100%}
  x={100}
  y={100}
>
  Rnd
</Rnd>
```

**Comment by bokuweb** [comment]
Please reopen this issue if you need.
